src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1854,6 +1854,45 @@ limitations under the License.
             <include cluster="Relative Humidity Measurement" client="false" server="false" clientLocked="false" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-fan</name>
+        <domain>HA</domain>
+        <typeName>Matter Fan</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x003B</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+                <requireCommand>IdentifyQuery</requireCommand>
+                <requireCommand>TriggerEffect</requireCommand>
+            </include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
+                <requireAttribute>SERVER_LIST</requireAttribute>
+                <requireAttribute>CLIENT_LIST</requireAttribute>
+                <requireAttribute>PARTS_LIST</requireAttribute>
+            </include>
+            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="true">
```

Comment:
Why is the binding added here? Both server and client are set to false and locked so why are they required here?

Reply:
Removed